display_list/display_list_image.h

```diff
@@ -81,6 +86,16 @@ class DlImage : public SkRefCnt {
   ///
   SkIRect bounds() const;
 
+  //----------------------------------------------------------------------------
+  /// @return     Whether the image method is only safe to use on the raster
```

Comment:
The owning context of this image, not really whether. the owning context describes where this image is safe to use. though maybe that last part could be more defined

Reply:
Done